relay/implementations/publish_evm_transaction.ts

```diff
@@ -71,6 +72,32 @@ export async function publishTransaction(
         });
       }
 
+      switch (chainId) {
```

Comment:
Here you can use the **data-driven programming** approach as @PetarKirov suggested

But it can be done in a following PR.